.github/workflows/pytest-pr.yml

```diff
@@ -55,7 +55,7 @@ jobs:
       - name: Install pytest
         run: |
           pip install -e ${{ matrix.extras }}
-          pip install pytest pytest-recording pytest-asyncio pytest-cov
```

Comment:
Depends on how strict polls is, but technically this does not belong in this PR

Reply:
Not feeling strict at all

Reply:
I included it because I didn't want to add typeguard to both files ;)